src/main/java/hanieum/conik/adapter/company/persistence/CompanyEntity.java

```diff
@@ -0,0 +1,53 @@
+package hanieum.conik.adapter.company.persistence;
+
+import hanieum.conik.domain.company.CompanyStatus;
+import hanieum.conik.global.domain.BaseEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "company")
+@Data
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class CompanyEntity extends BaseEntity {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "company_name", nullable = false)
+    private String name;
+
+    @Column(name = "company_owner", nullable = false)
+    private String owner;
+
+    @Column(name = "company_owner_email", nullable = false)
+    private String ownerEmail;
+
+    @Column(name = "company_owner_phone", nullable = false)
+    private String ownerPhoneNumber;
+
+    @Column(name = "company_business_type", nullable = false)
+    private String businessType;
+
+    @Column(name = "company_industry", nullable = false)
+    private String industry;
+
+    @Column(name = "company_registration_number", nullable = false)
+    private String registrationNumber;   // camelCase
+
+    @Column(name = "company_registration_certificate_url", nullable = false)
+    private String registrationCertificateUrl;
+
+    @Column(name = "company_bank_account_url", nullable = false)
+    private String bankAccountUrl;
+
+    @Column(name = "company_profile_url", nullable = false)
+    private String profileUrl;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "company_status", nullable = false)
+    private CompanyStatus status;
```

Comment:
저는 개인적으로 나중에 변수를 불러쓰거나 할 때 같은 엔티티속성명이 존재하면 헷갈리는 일이 많아서 prefix로 엔티티명을 사용하는 데 어떻게 생각하시나요?
참고로 자동으로 컬럼명은 스네이크 케이스로 만들어줘서, 같은 걸로 하고 싶다면 엔티티속성명은 지정하지 않아도 돼요
```suggestion
@Enumerated(EnumType.STRING) 
@Column(nullable = false)
private CompanyStatus companyStatus
```

Reply:
변수를 불러쓸때 company.getCompanyStatus() 이렇게 하면 변수명이 길어지거나 할때 코드가 너무 길어지는 것 같다고 생각해서 저는 company.getStatus() 만으로도 의미가 잘 전달된다고 생각해서 계속 저렇게 작성하기는 했습니다.
헷갈리는 경우 방지하려면 prefix로 해도 좋은 것 같아요. 그렇게 통일할까요?

Reply:
prefix 빼고하는걸로 결정!